### PR TITLE
[SERVDEV-349] elements collector directive

### PR DESCRIPTION
**Pull request by davidmsantos90**
- Added sorting to the elements collector directive
 - Jasmine unit tests for the directive

@graimundo please review

### CODE REVIEW

File: pentaho-det-core/src/main/resources/web/index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     <title>Data Explorer Tool</title>
     <!-- Bootstrap CSS -->
     <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/2.3.1/css/bootstrap.min.css" rel="stylesheet">
-    <link href="./pentaho-det.css" rel="stylesheet">
+    <link href="/DataExplorerModule/web/data-explorer-module.css" rel="stylesheet">
```
Comment (graimundo) [inline pentaho-det-core/src/main/resources/web/index.html]: Turn into relative path.

File: data-explorer-module/src/test/resources/requireConfig.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+require.config({
+    "paths": {
+        "jquery": "/webjars/jquery/2.1.4/jquery",
```
Comment (graimundo) [inline data-explorer-module/src/test/resources/requireConfig.js]: Can't these paths be made relative?

Comment (nantunes) [inline data-explorer-module/src/test/resources/requireConfig.js]: "/webjars" is a special prefix so maven-jasmine plugin retrieves the script from webjar.

File: pentaho-det-core/src/main/resources/web/index.html (hunk 1 above)
Comment (nantunes) [inline pentaho-det-core/src/main/resources/web/index.html]: It shouldn't be here at all. The core can't be aware of module's styles (or any other resource).

We must concebe a way to inject those at runtime.

**Comment by nantunes** [comment]
The PR is valid, as it doesn't break anything, so I'll merge and resolve the commented issues in a future PR.

**Pull request merged**